bin/genesis

```diff
@@ -1497,7 +1533,7 @@ sub parse_pipeline {
 	my $src = $pipeline->{pipeline}{layouts}{$layout}
 		or die "No such layout `${layout}'\n";
 
-	# our internal representatio
+	# our internal representation
```

Comment:
Is this relevant to the addition of a `ci` sub-command?

Reply:
Not really directly, it is one tiny typo I noticed, so I just fixed it without a separate PR